### DISCUSSION TITLE
Welcome to spec-parser Discussions!

### DISCUSSION THREAD

**Discussion by zvr** [spdx/spec-parser#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below or open another discussion thread and tell us about ideas for this project.

**Comment by nirmal-suthar** [comment]
Posting some relevant docs and links (from Project page):
- [Discussion](https://docs.google.com/document/d/13PojpaFPdoKZ9Gyh_DEY-Rp7lldyMbSiGE3vCRQhR9M/edit#heading=h.rg0y96899jlf)
- [Markdown specs](https://docs.google.com/document/d/1LN5CepVVOu38w4pXeLpw_3BDNn2CKAWUyTVdlh8C2WM/edit)
- [Analysis of RDF/OWL fields relative to Markdown](https://docs.google.com/document/d/1EGoAmKxPfmmlF3XV6fXwNmsCiFKLH83Bhh8_xrmGhko/edit)

**Reply by nirmal-suthar** [reply]
@zvr I was going through markdown specs (which I believe is in WIP). There are some fields which are different like in specs table is specified in Metadata section while in md files tables are Description section. So wanted to know if there is plan to change or already changed.

**Reply by zvr** [reply]
that's what we nave to clear up...

**Comment by nirmal-suthar** [comment]
@zvr How one will differentiate between File.md and FileType.md? Is it the suffix `*Type.md` ?
